runtime-modules/content/src/lib.rs

```diff
@@ -1031,54 +962,6 @@ decl_module! {
 
             Self::deposit_event(RawEvent::VideoCategoryDeleted(actor, category_id));
         }
-
```

Comment:
Isn't it a rebase artifact?

Reply:
I am not following. The original code in olympia has this code. In this PR that I am opening I removed it

Reply:
Ok. So, it's part of the PR. I was expecting only the NFT-related code.